.gitignore

```diff
@@ -7,6 +7,9 @@
 ##
 ## Get latest from https://github.com/github/gitignore/blob/main/VisualStudio.gitignore
 
+#AppSettings
+appsettings.json
```

Comment:
We will discuss it if necessary on discord, but appsettings need to be pushed due to Docker needs